### PR TITLE
Make iOS ECC Wallet localizable

### PR DESCRIPTION
**Pull request by yacare555**
Hi, I'm just starting with gitcoin.
It adds localization functionality and also the Spanish translation.
Hope it helps you.

### CODE REVIEW

**Comment by pacu** [comment]
Hola, gracias por tu PR. Voy a estar mirándolo lo más pronto que pueda.

File: wallet/wallet/es.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+"Transaction Detail" = "Detalles de la tx";
+
+//EnterRecipient
+"Enter a shielded Zcash address" = "Escribe una direccion blindada de Zcash";
+"This is a valid Zcash address!" = "Es una dirección Zcash válida!";
+"Invalid Zcash address!" = "Dirección Zcash no válida!";
+"You Have %@ sendable ZEC" = "Tienes %@ ZEC para enviar";
+"%@ sendable ZEC. You don't have sufficient funds to cover the amount + Miner Fee of %@ ZEC" = "%@ ZEC para enviar. No tiene fondos suficientes para cubrir esa cantidad + la tarifa de minero de %@ ZEC";
+"You don't have any sendable ZEC yet" = "Todavía no tienes ningún ZEC para enviar";
+"Zcash address in buffer" = "Dirección de Zcash en el búfer";
```
Comment (pacu) [inline wallet/wallet/es.lproj/Localizable.strings:52]: change "búfer" to "portapapeles"

File: wallet/wallet/es.lproj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+"Next" = "Next";
+
+"Amount" = "Amount";
+
+//AddMemo
+"Add Memo" = "Agregar Memo";
+"Your address is shielded from the public,\n but will be available to the receipient via the memo field." = "Su dirección está protegida del público, \n pero estará disponible para el destinatario a través del campo memo.";
```
Comment (pacu) [inline wallet/wallet/es.lproj/Localizable.strings:59]: change to "Su dirección está protegida y no será pública, pero estará disponible para el destinatario a través del memo.

File: wallet/wallet/es.lproj/Localizable.strings [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+"Your transaction is shielded and your address is unavailable to receipent." = "Su transacción está protegida y su dirección no está disponible para el destinatario.";
```
Comment (pacu) [inline wallet/wallet/es.lproj/Localizable.strings:60]: "Su transacción esta blindada y su dirección no estará disponible para el destinatario"

File: wallet/wallet/es.lproj/Localizable.strings [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+
+
+//CreateNewWallet
+"Create New" = "Crear nueva";
+"Restore" = "Restaurar";
+
+//RestoreWallet
+"Enter your Seed Phrase" = "Ingrese su frase semilla (Seed)";
+"Make sure nobody is watching you!" = "¡Asegúrate de que nadie te esté mirando!";
+"Wallet Birthday height" = "Talla de cumpleaños de cartera";
```
Comment (pacu) [inline wallet/wallet/es.lproj/Localizable.strings:87]: this is a tough one. Conceptually, birthday can mean "day of birth" or "the event of day of birth remembrance". In spanish "cumpleaños" means the latter more than the former, so the "cumpleaños de cartera" doesn't make any sense for a native Spanish speaker. I'd go for "Bloque de nacimiento", which really entails what "wallet birthday height" means, which is at which height your wallet was "born". "Numero de bloque de nacimiento" is conceptually more accurate.

File: wallet/wallet/es.lproj/Localizable.strings [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+"If you don't know it, leave it blank. First Sync will take longer." = "Si no lo sabe, déjelo en blanco. La primera sincronización llevará más tiempo.";
+"Proceed" = "Continuar";
+"Restore from Seed Phrase" = "Restaurar de la frase de semilla";
```
Comment (pacu) [inline wallet/wallet/es.lproj/Localizable.strings:90]: propositions are wrong change "Restaurar de la frase de semilla" -> to "Restaurar sede frase semilla"

File: wallet/wallet/es.lproj/Localizable.strings [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+
+//BackupWallet
+"Your Wallet needs\nto be Backed up" = "Su cartera necesita \n ser respaldada";
+"Syncing %@%" = "Sincronizando %@%";
+"Skip" = "Saltar";
+
+//SeedBackup
+"Seed:" = "Semilla(Seed):";
+"Wallet Birthday:" = "Cumpleaños de la Cartera:";
```
Comment (pacu) [inline wallet/wallet/es.lproj/Localizable.strings:99]: see comment above

File: wallet/wallet/es.lproj/Localizable.strings [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+//RestoreWallet
+"Enter your Seed Phrase" = "Ingrese su frase semilla (Seed)";
+"Make sure nobody is watching you!" = "¡Asegúrate de que nadie te esté mirando!";
+"Wallet Birthday height" = "Talla de cumpleaños de cartera";
+"If you don't know it, leave it blank. First Sync will take longer." = "Si no lo sabe, déjelo en blanco. La primera sincronización llevará más tiempo.";
+"Proceed" = "Continuar";
+"Restore from Seed Phrase" = "Restaurar de la frase de semilla";
+
+//BackupWallet
+"Your Wallet needs\nto be Backed up" = "Su cartera necesita \n ser respaldada";
```
Comment (pacu) [inline wallet/wallet/es.lproj/Localizable.strings:93]: don't translate wallet

File: wallet/wallet/es.lproj/Localizable.strings [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+"Syncing %@%" = "Sincronizando %@%";
+"Skip" = "Saltar";
+
+//SeedBackup
+"Seed:" = "Semilla(Seed):";
+"Wallet Birthday:" = "Cumpleaños de la Cartera:";
+"Your Backup Seed" = "Su semilla (Seed) de respaldo";
+"Please back them up wisely!\nWe recommend a paper backup and a password vault" = "¡Haz una copia de seguridad sabiamente! \n Recomendamos una copia de seguridad en papel y una bóveda de contraseñas";
```
Comment (pacu) [inline wallet/wallet/es.lproj/Localizable.strings:101]: change "bóveda de contraseñas" to "administrador de contraseñas"

File: wallet/wallet/es.lproj/Localizable.strings [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+"Copy to clipboard" = "Copiar al portapapeles";
+"I'm all set!" = "¡Estoy listo!";
+
+//Home
+"There was an error creating your wallet. Please back it up and try again" = "Se produjo un error al crear tu cartera. Por favor, haga una copia de seguridad e intente nuevamente";
```
Comment (pacu) [inline wallet/wallet/es.lproj/Localizable.strings:106]: change "Se produjo un error al crear tu cartera. Por favor, haga una copia de seguridad e intente nuevamente" to "Se produjo un error al crear tu wallet. Por favor, haga una copia de seguridad de su frase semilla e intente nuevamente";

File: wallet/wallet/es.lproj/Localizable.strings [hunk 10]
```diff
@@ -0,0 +1,156 @@
...
+"Max Retry attempts (%@) have been reached" = "Ha alcanzado el límite máximo de reintentos (%@)";
+"dismiss" = "descartar";
+"Retry" = "Reintentar";
+"Send" = "Enviar";
+"No Balance" = "Sin Balance";
+"Wallet History" = "Historia de la Cartera";
```
Comment (pacu) [inline wallet/wallet/es.lproj/Localizable.strings:112]: "historial de wallet"

File: wallet/wallet/es.lproj/Localizable.strings [hunk 11]
```diff
@@ -0,0 +1,156 @@
...
+//ShareScreen
+"Share Screen" = "Compartir pantalla";
+
+//ProfileScreen
+"Send Feedback" = "Enviar comentarios";
+"See Application Log" = "Ver log de la aplicación";
+"ECC Wallet" = "ECC Wallet";
+
+//FundsAtRisk
+"Remember, with Zcash YOU are the bank. Only you, or anyone with your seed phrase, has access to your wallet." = "Recuerde, con Zcash USTED es el banco. Solo usted, o cualquier persona con su frase inicial, tiene acceso a su cartera.";
```
Comment (pacu) [inline wallet/wallet/es.lproj/Localizable.strings:135]: change from "Usted" to second person "Tú"

File: wallet/wallet/es.lproj/Localizable.strings [hunk 12]
```diff
@@ -0,0 +1,156 @@
...
+"as no one else can restore it for you" = "ya que nadie más puede restaurarlo por ti";
+"if your device is lost/broken/stolen…" = "si su dispositivo se pierde / se rompe / es robado…";
+"Your " = "Tus ";
+"funds " = "fondos ";
+"are at risk!" = "están en peligro!";
+"Backup now!" = "Respaldar ahora!";
+"Not now" = "Ahora no";
+
+//NukeWarning
+"Make sure you backed up your wallet before proceeding." = "Asegúrese de hacer una copia de seguridad de su cartera antes de continuar.";
```
Comment (pacu) [inline wallet/wallet/es.lproj/Localizable.strings:146]: change "seguridad" to "respaldo"

File: wallet/wallet/es.lproj/Localizable.strings [hunk 13]
```diff
@@ -0,0 +1,156 @@
...
+"If you nuke your wallet, there's no way to recover it." = "Si destruyes tu cartera, no hay forma de recuperarla.";
```
Comment (pacu) [inline wallet/wallet/es.lproj/Localizable.strings:147]: don't translate wallet

File: wallet/wallet/es.lproj/Localizable.strings [hunk 14]
```diff
@@ -84,32 +84,32 @@
...
 "If you don't know it, leave it blank. First Sync will take longer." = "Si no lo sabe, déjelo en blanco. La primera sincronización llevará más tiempo.";
 "Proceed" = "Continuar";
-"Restore from Seed Phrase" = "Restaurar de la frase de semilla";
+"Restore from Seed Phrase" = "Restaurarse de frase semilla";
 
 //BackupWallet
-"Your Wallet needs\nto be Backed up" = "Su cartera necesita \n ser respaldada";
+"Your Wallet needs\nto be Backed up" = "Su wallet necesita \n ser respaldada";
 "Syncing %@%" = "Sincronizando %@%";
 "Skip" = "Saltar";
```
Comment (pacu) [inline wallet/wallet/es.lproj/Localizable.strings:95]: Oh, I just saw this: Skip correct translation is "Omitir"

**Comment by pacu** [comment]
@yacare555 are you done with your changes? let me know so I can do the review.

**Comment by yacare555** [comment]
Hi Pacu, sorry I forget request review. I'm done.

**Comment by pacu** [comment]
@yacare555  could you fix the merge conflicts?

**Comment by pacu** [comment]
@yacare555 It was only one line, let's see if ci passes

**Comment by yacare555** [comment]
Great, thanks!

**Pull request merged**